src/pcap/pcap_test.go

```diff
@@ -0,0 +1,97 @@
+package pcap
+
+import (
+	"testing"
+)
+
+func TestCaptureOptions_validate(t *testing.T) {
+	tests := []struct {
+		name    string
+		opts    *CaptureOptions
+		wantErr bool
+	}{
+		{
+			name:    "Empty device name",
+			opts:    &CaptureOptions{Filter: "host 10.0.0.1", SnapLen: 65000},
+			wantErr: true,
+		},
+		{
+			name:    "Invalid device name",
+			opts:    &CaptureOptions{Device: "eth0-=", Filter: "host 10.0.0.1", SnapLen: 65000},
```

Comment:
I don't think there is a rule against calling your NIC `eth0.4` for VLANs, and probably you could have some that have a dash, e.g. `network-4`. I couldn't find hard and fast rules what Linux would accept as device names, but certainly it's not just alphanumeric.

Reply:
Discussion will continue in the [comment above](https://github.com/cloudfoundry/pcap-release/pull/9#discussion_r1053368868)